src/attribute.rs

```diff
@@ -408,7 +410,33 @@ impl<'a> Attribute<'a> {
                 Ok(AttrValue::Str(String::from(String::from_utf8_lossy(
                     &buf[..pos],
                 ))))
-            }
+            },
+            NC_STRING => {
+                let mut buf: Vec<*mut c_char> = vec![std::ptr::null_mut(); attlen];
+                let mut result: Vec<String> = Vec::with_capacity(attlen);
+                unsafe {
+                    error::checked(super::with_lock(|| {
+                        nc_get_att_string(
+                            self.ncid,
+                            self.varid,
+                            self.name.as_ptr() as *const _,
+                            buf.as_mut_ptr() as *mut *mut _,
+                        )
+                    }))?;
+
+                    for str_ptr in &buf {
+                        if str_ptr.is_null() {
+                            break;
+                        }
+                        let c_str = CStr::from_ptr(*str_ptr);
+                        result.push(c_str.to_string_lossy().to_string());
+                    }
+                    nc_free_string(attlen, buf.as_mut_ptr());
+                }
+                Ok(AttrValue::Str(
+                    result.join("\n")
```

Comment:
This is an assumption. Do we want to have a separate `AttrValue` variant?

Reply:
We should add `Strs(Vec<String>)` here, since a string attribute can consist of multiple `nc_string`

Reply:
Implemented. Please, review.